docs/jwst/skymatch/description.rst

```diff
@@ -0,0 +1,247 @@
+results computed by each method (i.e. the values of the resulting BKGLEVEL keywords).
+All results are for the case where the step argument ``match_down = True``,
+which means matching is done to the image with the lowest sky value.
+Note that these examples are for the highly simplistic case where each example
+image contains nothing but the constant sky value. Hence the sky computations
+are not affected at all by any source content and are therefore able to
+determine the sky values exactly in each image. Results for real images will
+of course not be so exact.
+
++-------+-------+--------+-------+--------------+
+| Sky   | Local | Global | Match | Global+Match |
++=======+=======+========+=======+==============+
+| 100   |  100  |  100   |    0  |        100   |
++-------+-------+--------+-------+--------------+
+| 120   |  120  |  100   |   20  |        120   |
++-------+-------+--------+-------+--------------+
+| 105   |  105  |  100   |    5  |        105   |
++-------+-------+--------+-------+--------------+
+| 110   |  110  |  100   |   10  |        110   |
++-------+-------+--------+-------+--------------+
+| 105   |  105  |  100   |    5  |        105   |
++-------+-------+--------+-------+--------------+
+| 115   |  115  |  100   |   15  |        115   |
++-------+-------+--------+-------+--------------+
+
+- "local" finds the sky level of each image independently of the rest.
+- "global" uses the minimum sky level found by "local" and applies it to all images.
+- "match" with "match_down=True" finds the offset needed to match all images
+  to the level of the image with the lowest sky level.
+- "global+match" with "match_down=True" finds the offsets and global value
+  needed to set all images to a sky level of zero. In this trivial example,
+  the results are identical to the "local" method.
+
+The second example is for a set of 7 images, where the first 4 form a 2x2
+mosaic, with overlaps, and the second set of 3 images forms another mosaic,
+with internal overlap, but the 2 mosaics do *NOT* overlap one another.
+
++-------+-------+--------+-------+--------------+
+| Sky   | Local | Global | Match | Global+Match |
++=======+=======+========+=======+==============+
+| 100   |  100  |   90   |     0 |    86.25     |
++-------+-------+--------+-------+--------------+
+| 120   |  120  |   90   |    20 |   106.25     |
++-------+-------+--------+-------+--------------+
+| 105   |  105  |   90   |     5 |    91.25     |
++-------+-------+--------+-------+--------------+
+| 110   |  110  |   90   |    10 |    96.25     |
++-------+-------+--------+-------+--------------+
+|  95   |   95  |   90   |  8.75 |     95       |
++-------+-------+--------+-------+--------------+
+|  90   |   90  |   90   |  3.75 |     90       |
++-------+-------+--------+-------+--------------+
+| 100   |  100  |   90   | 13.75 |    100       |
++-------+-------+--------+-------+--------------+
+
```

Comment:
Thanks for this example. I am a bit puzzled by the results of the three images at the bottom. Why is the result for "match" showing decimal values as well? Also, why is the result of "global+match" showing the same results for local for the last three images (as it should) but strange values for the top  four images?